src/main/frontend/src/components/aggregates/AggregateDialog.svelte

```diff
@@ -71,7 +71,7 @@
 
 	const reset = () => {
 		aggregateName = "";
-		stateFields = [{ name: "id", type: "String" }];
+		stateFields = [{ name: "id", type: "String", removable: false }];
```

Comment:
Have you checked if this is still working with the generator? (If not, you need to filter it out before pressing **Generate**, which was my concern, so I just went the easy route for now)

Reply:
fixed